.github/workflows/ci.yml

```diff
@@ -8,10 +8,15 @@ on:
 jobs:
   test:
     runs-on: ${{ contains( github.event.pull_request.labels.*.name, 'use_fast_ci') && 'ubuntu-latest-8core' || 'ubuntu-latest' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
```

Comment:
The word `target` is arbitrary here, just something to reference later as `${{ matrix.target }}`